redaxo/src/addons/media_manager/lib/media_manager.php

```diff
@@ -257,7 +257,11 @@ public function getCacheFilename()
     {
         assert(null !== $this->cachePath);
         assert(null !== $this->type);
-        return $this->cachePath . $this->type . '/' . $this->originalFilename;
+
+        $cachePath = rex_extension::registerPoint(new rex_extension_point('MEDIA_MANAGER_CACHE_FILENAME_PATH', $this->cachePath, ['object' => $this,
```

Comment:
type und file würde ich ruhig zusätzlich als Params liefern. Und den Param mit `$this` würde ich `manager` nennen.

```suggestion
        $cachePath = rex_extension::registerPoint(new rex_extension_point('MEDIA_MANAGER_CACHE_FILENAME_PATH', $this->cachePath, [
            'manager' => $this,
            'type' => $type,
            'file' => $this->originalFilename,
```